diff --git a/arch/arm/boot/dts/stm32mp157a-dk1.dts b/arch/arm/boot/dts/stm32mp157a-dk1.dts
index 5a79915d1..03ef8c965 100644
--- a/arch/arm/boot/dts/stm32mp157a-dk1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-dk1.dts
@@ -211,6 +211,8 @@
                 compatible = "invensense,mpu9250";
                 reg = <0x68>;
                 status = "okay";
+                interrupt-parent = <&gpioe>;
+                interrupts = <12 IRQ_TYPE_LEVEL_HIGH>;
         };
 };
 
